lib/Core/Memory.cpp

```diff
@@ -28,16 +28,72 @@
 #include <cassert>
 #include <sstream>
 
-using namespace llvm;
 using namespace klee;
 
 namespace {
-  cl::opt<bool>
-  UseConstantArrays("use-constant-arrays",
-                    cl::desc("Use constant arrays instead of updates when possible (default=true)\n"),
-                    cl::init(true),
-                    cl::cat(SolvingCat));
+llvm::cl::opt<bool>
+    UseConstantArrays("use-constant-arrays",
+                      llvm::cl::desc("Use constant arrays instead of updates "
+                                     "when possible (default=true)\n"),
+                      llvm::cl::init(true), llvm::cl::cat(SolvingCat));
+}
+
+#if LLVM_VERSION_CODE < LLVM_VERSION(9, 0)
```

Comment:
These functions should work with newer LLVM versions too AFAIK, the storage representation of APInt hasn't changed since.